model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -329,6 +331,35 @@ object WorkspaceShardStates {
   case object Unknown extends WorkspaceShardState
 }
 
+object WorkspaceType {
+  /**
+    * Represents the system-of-record for a workspace
+    * rawls workspaces are owned and managed by rawls
+    * mc workspaces are owned and managed by workspace manager
+    */
+  sealed trait WorkspaceType extends RawlsEnumeration[WorkspaceType] {
+    override def toString: String = WorkspaceType.toString(this)
+    override def withName(name: String): WorkspaceType = WorkspaceType.withName(name)
+  }
+
```

Comment:
@aherbst-broad is it feasible to add test coverage for these lines?

Reply:
Good call, [done](https://github.com/broadinstitute/rawls/pull/1629/commits/33787a8f402e4acb36c6730a975af7fbafc156c0)